python/pyarrow/array.pxi

```diff
@@ -4343,6 +4343,30 @@ cdef class ExtensionArray(Array):
         return result
 
 
+class JsonArray(ExtensionArray):
+    """
+    Concrete class for Arrow arrays of JSON data type.
+
```

Comment:
Should we maybe note here that this array class does not guarantee that the data is actually valid JSON? (which I assume is the case?)

For example in the code example below, you can in theory put whatever string data you want in the array.

Reply:
Added a not. Let me know if it's not extensive enough.